packages/web/src/components/user-list/utils.ts

```diff
@@ -22,7 +22,7 @@ export type UserListProviderArgs<T> = {
     offset: number
     entityId: ID
     currentUserId: ID | null
-  }) => Promise<UserMetadata[]>
+  }) => Promise<{ users: UserMetadata[]; extra?: any }>
```

Comment:
any chance we can type extra? or maybe use `unknown`?

Reply:
Maybe could be another generic type param?